src/rarity/rarity.meta.ts

```diff
@@ -243,6 +243,40 @@ const ensMetaFunc = (nftTraits: TraitBase[], collection: EnsCollectionSlug) => {
         }
     }
 
+    // Hundred (x00), Thousand (x000), Ten Thousand (x0,000) trait
+    switch (digits) {
+        case 3:
+            if (id % 100 === 0) {
+                outTraits.push({
+                    value: 'Hundred',
+                    category: 'Meta',
+                    typeValue: 'Special',
+                    displayType: null,
+                });
+            }
+            break;
+        case 4:
+            if (id % 1000 === 0) {
+                outTraits.push({
+                    value: 'Thousand',
+                    category: 'Meta',
+                    typeValue: 'Special',
+                    displayType: null,
+                });
+            }
+            break;
+        case 5:
+            if (id % 10_000 === 0) {
+                outTraits.push({
+                    value: 'Ten Thousand',
+                    category: 'Meta',
+                    typeValue: 'Special',
+                    displayType: null,
+                });
+            }
+            break;
```

Comment:
we could always do something like `id % Math.pow(10,digits)` to check if it's valid, and have a dict of digits:trait_value, to cut this down, but it's not a blocking concern

Reply:
digits-1*

Reply:
I thought about doing that, but the dict initially turned me off -- happy to go for it

Reply:
i like it as a way to have this be as succinct as possible